src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vedtak/VedtakSteg.kt

```diff
@@ -44,24 +44,19 @@ class VedtakSteg(
         val behandling = behandlingService.hentBehandling(behandlingId)
         validerAtBehandlingErGyldigForVedtak(behandling)
 
-        if (behandling.skalBehandlesAutomatisk()) {
-            loggService.opprettSendTilBeslutterLogg(behandling.id)
```

Comment:
Ettersom vi ikke skal utføre vedtaksteget for automatiske behandlinger, trenger vi ikke lenger denne løypen